tests/integration/submissions/test_submissions_views.py

```diff
@@ -203,6 +228,33 @@ def test_new_literature_submit(
     }
 
 
+@patch("inspirehep.submissions.views.requests.post")
+def test_new_literature_submit_with_session_user(
+    mock_requests_post, app, api_client, create_user
+):
+    mock_requests_post.return_value.status_code = 200
+    mock_requests_post.return_value.content = jsonify({"workflow_object_id": 30})
+    user = create_user()
+    login_user_via_session(api_client, email=user.email)
+    response = api_client.post(
+        "/submissions/literature",
+        content_type="application/json",
+        data=json.dumps(
+            {
+                "data": {
+                    "document_type": "article",
+                    "authors": [{"full_name": "Urhan, Harun"}],
+                    "title": "Discovery of cool stuff",
+                    "subjects": ["Other"],
+                    "pdf_link": "https://cern.ch/coolstuff.pdf",
```

Comment:
can you remove this? since it's just testing for auth, this data is unnecessary.